### ISSUE TITLE
Height and Width 100px?

### ISSUE THREAD

**Issue by TheDevelolper** [goldsam/ng-golden-layout#3]
Hey I'm using your lib. I have it rendering only it insists on being 100px by 100px. Is there a build in way to change the sizing to 100% by 100%?

**Comment by TheDevelolper** [comment]
this is what seems to get rendered: 
`<div class="lm_goldenlayout lm_item lm_root" style="width: 100px; height: 100px;">`

**Comment by TheDevelolper** [comment]
I've tried everything... even using jquery to change the width and height. I've tried running the example app but I just get other errors.

**Comment by goldsam** [comment]
I don't believe there is any code in my library which should explicitly produce a width/height of exactly 100px. Do you have any code which demonstrates the issue?

**Comment by Arnie38** [comment]
I use 100% for width and height just fine within the content of a golden layout panel. It works as expected.
